spec/models/user_spec.rb

```diff
@@ -0,0 +1,12 @@
+require 'rails_helper'
+
+RSpec.describe User, type: :model do
+	it "evaluates total purchased for user" do
+		user = create(:user)
+		order_1 = create(:order, user_id: user.id, total_price: 5000)
+		order_2 = create(:order, user_id: user.id, total_price: 1500)
+		order_3 = create(:order, user_id: user.id, total_price: 900)
```

Comment:
Useless assignment to variable - `order_3`. Did you mean `order_1`?<br>Tab detected.